### PR TITLE
Update cyclical_learning_rate.py

### PR DESCRIPTION
**Pull request by DragonPG2000**
# Description

Brief Description of the PR: Updating the documentation of cyclic learning rate to describe the step_size argument in simpler terms. 

Fixes # (issue)
The issue was raised on TF-Addons:  https://github.com/tensorflow/addons/issues/2181

### CODE REVIEW

**Comment by DragonPG2000** [comment]
@googlebot I signed it!

**Review by seanpmorgan — CHANGES_REQUESTED** [review]
Thanks @DragonPG2000 ! The original issues lists 3 learning rates that need documentation updates. Mind adding the other two please.

**Comment by DragonPG2000** [comment]
It's done.

**Review by seanpmorgan — APPROVED** [review]
Sorry for the slow response. LGTM Thanks!
